packages/tupaia-web/src/api/queries/useEntityAncestors.ts

```diff
@@ -4,14 +4,13 @@
  */
 
 import { useQuery } from 'react-query';
-import { TupaiaWebEntitiesRequest } from '@tupaia/types';
-import { EntityCode, ProjectCode } from '../../types';
+import { Entity, EntityCode, ProjectCode } from '../../types';
 import { get } from '../api';
 
 export const useEntityAncestors = (projectCode?: ProjectCode, entityCode?: EntityCode) => {
   return useQuery(
     ['entityAncestors', projectCode, entityCode],
-    (): Promise<TupaiaWebEntitiesRequest.ResBody> =>
+    (): Promise<Entity[]> =>
       get(`entityAncestors/${projectCode}/${entityCode}`, { params: { includeRootEntity: true } }),
```

Comment:
More a curiosity thing, not to challenge this change:

Why make this change here, rather than updating `TupaiaWebEntitiesRequest.Resbody` not to use `KeysToCamelCase<>`?

Reply:
Just because elsewhere we use the results of this in the same sort of way we use the results of `useEntities` and so it was simpler to make them the same types